packages/swap/prisma/migrations/20240216112846_change_asset_values/migration.sql

```diff
@@ -0,0 +1,8 @@
+-- AlterEnum
+BEGIN;
```

Comment:
```suggestion
```

Reply:
is this just for performance or is there a problem with doing it in a tx?

Reply:
i think its good to leave them here in case something goes wrong mid migration

Reply:
migrations are already wrapped in transactions by prisma